src/esp/bindings/SensorBindings.cpp

```diff
@@ -21,6 +21,10 @@
 #ifdef ESP_BUILD_WITH_CUDA
 #include "esp/sensor/RedwoodNoiseModel.h"
 #endif
+// todo sangarg : check flag
```

Comment:
QUESTION: I'm not sure why the ESP_BUILD_WITH_AUDIO flag is not getting set. Could not figure out the correct cmakelist.txt to set the flag. Please let me know how to fix this.

Reply:
The flag is propagated to `esp/core/configure.h`, so if you include that file, you should get the flag.